docs/building.md

```diff
@@ -528,6 +528,44 @@ Executing `Packaging/miyoo_mini/build.sh` will create the folder `build-miyoo-mi
 OnionOS Port Collection.
 </details>
 
+<details><summary>macOS 10.4 Tiger</summary>
+
+For macOS Tiger, DevilutionX can be compiled using the compiler and libraries from [MacPorts](https://www.macports.org/).
+
+For PowerPC, you can use precompiled dependencies from here:
+
+http://macports-tiger-ppc.glebm.com/
+
+After installing MacPorts, run:
+
+~~~ bash
+# Some packages may require you to manually deactivate certain ports during installation.
+# Remember to reactivate them after installing.
+sudo port install curl curl-ca-bundle gcc14 cmake \
+  libsdl12 libsdl_image libsodium libfmt11 bzip2 zlib lua54 \
+  google-benchmark gtest
```

Comment:
gtest appears to be disabled as well

Reply:
... also google-benchmark isn't in my ports yet :)

Reply:
Neither google-benchmark nor gtest work from macports (google-benchmark has been merged though), I'll push a commit to use the vendored ones in a moment